Makefile

```diff
@@ -126,9 +126,15 @@ dune: $(BIN)
 	$(BIN) $(RUN_ARGS)
 
 .PHONY: default install uninstall reinstall clean test doc dev-switch
-.PHONY: promote accept-corrections opam-release dune check fmt
+.PHONY: promote accept-corrections release opam-release dune check fmt
 
-opam-release:
+# Use this target to make sure that we always run the in source dune when making
+# the release
+release: dev
```

Comment:
There is already a `release` target in this Makefile

Reply:
Ah damn. I reverted the name to `opam-release`.